package.json

```diff
@@ -245,13 +245,18 @@
   "scripts": {
     "lint": "tslint './src/**/*.ts' './test/**/*.ts'",
     "pretty": "tslint --fix './src/**/*.ts' './test/**/*.ts'",
+	"pre-commit": "lint-staged",
     "test": "npm run compile && node ./out/test/runTest.js",
     "test-unit": "cross-env MOCHA_GREP=@unit npm test --silent",
     "test-integration": "cross-env MOCHA_GREP=@integration npm test --silent",
     "vscode:prepublish": "tsc -p ./",
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
-    "build": "npm install && npm run lint && npm run compile"
+    "build": "npm install && npm run lint && npm run compile",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+	"{src,test}/**/*.ts": [ "tslint --fix" ]
```

Comment:
[F] Lifted from https://github.com/typicode/husky/issues/949#issuecomment-821917498

Reply:
makes sense, nice hook to get this into everyones commit flow.